index.d.ts

```diff
@@ -0,0 +1,3 @@
+// TypeScript Version: 3.4
+
+export function placeholderForDtslint(): void
```

Comment:
Why do we need this file?

Reply:
dtslint needs an `index.d.ts` file at the root of where it is called.
This exists purely to satisfy that constraint.

If there is an option to not require it, or an elegant want to call `dslint` for each package.
I'd be happy to remove it.

Reply:
I see.It seems the problem has been reported in here...!
https://github.com/microsoft/dtslint/issues/213

Reply:
I guess we don't have that much choice. It is needed for dtslint to determine supported typescript versions too.

Anyway, everything LGTM. 👍